### DISCUSSION TITLE
Preload warning in Next.js project

### DISCUSSION THREAD

**Discussion by olaj** [GoogleChrome/lighthouse#12103] [Q&A]
Not sure if this is the right forum but since lighthouse are throwing these warnings i suppose the problem can be debugged without having access to the code or understand next.js in detail.

I'm trying to improve the page speed score in my project and are getting pretty good numbers now. But this warning i get about preloaded links not being used has bugged me for a long time now.

The page i'm testing is https://alternativeto.net/software/firefox/ (i get the same issue on most of the pages on this site)

The same or similar issues has been discussed before.

https://github.com/vercel/next.js/issues/19525

I have linked some more issues in the last comment there.

This problem is reproducible in for example this blog starter i found:

https://github.com/JoseRFelix/nextjs-starter-blog

Just run build / start then run lighthouse and you get the warning. If you set all to prefetch false and run build / start again then you do not get the warning in lighthouse.
But i have tried to reproduce it with for example.

Thankful for any help / tips!

**Comment by patrickhulce** [comment]
Thanks @olaj for letting us know and for starting as a discussion :) 

This one is actually a bug in Lighthouse due to a recent change in Chrome's behavior about communicating whether a preload request was used or not. See https://github.com/GoogleChrome/lighthouse/issues/11933 for more details.

**Reply by olaj** [reply]
Ok, cool!

When will this be fixed then? I still get the warning in both page speed insights and in Lighthouse.

**Reply by patrickhulce** [reply]
There is no current timeline. Subscribe to #11933 for updates.
